Makefile

```diff
@@ -1,4 +1,4 @@
-GO=$(shell which go)
```

Comment:
Added this to allow using a custom go binary (useful when you have multiple Golang versions in your machine)

Reply:
Good thing